examples/codegen/README.md

```diff
@@ -0,0 +1,31 @@
+## End-to-end code generation example
+
+This example demonstrates an end-to-end pipeline of code generation and execution that typically occurs in a JIT engine.
+It creates a simple function (adding +1 to an array and storing the result in a second one) via llvm api,
+converts it to spirv, and submits it to the runtime.
+
+## Running the example
+
+### Using conda envrionment
+
+```
+$ conda env create -f third_party/deps.yml
+$ conda activate examples
+# configure with:
+$ cmake .. -DUR_BUILD_ADAPTER_L0=ON -DUR_BUILD_EXAMPLE_CODEGEN=ON
+$ make
+$ UR_ADAPTERS_FORCE_LOAD=./lib/libur_adapter_level_zero.so ./bin/codegen
+```
+
+### Without using conda
+
+To run the example with llvm 13, you will need to make sure that `llvm-13` and `libllvmspirvlib-13-dev` are installed.  
+
+**NOTE**: The example could be working with other llvm versions but it was tested with version 13.
+
+```
```

Comment:
Should these commands be executed in the current working dir or in the top-level/root repo dir?

The same question to the above commands - I can see you're using local dir `third_party/deps.yml`, but I believe CMake shouldn't be executed in this example's directory

Reply:
`third_party/deps.yml` should be only to initialize the conda environment and not being part of cmake.  I have added a clarification to note where this should be executed. let me know if it still have any ambiguity.

Reply:
ok, thanks for the clarification - no ambiguity :)

Nonetheless, I thought this `deps.yml` was a local file (in `examples` dir), but it's not.
Is it only prepared/tested with examples? Or should it work with all kinds of development/testing with Unified Runtime? // I'm asking because I'm not sure if we should add some kind of comment there, perhaps...?

Reply:
atm, it is used only to run the codegen example, but theoretically, I guess it should work if we use it to initialize any conda environment to run unified runtime, however i think it have a lot of extra packages that only codegen uses but UR doesn't need. I think it is better to add this comment for now and if the need differ for this file in the future we could change that comment.

Reply:
I added a comment at the beginning of the file to pinpoint that.